code/modules/mob/living/simple_animal/hostile/faithless.dm

```diff
@@ -22,7 +22,6 @@
 
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
-	speed = 4

```

Comment:
speed is incorrectly defined twice on faithless, for some reason; correct speed is `0`.